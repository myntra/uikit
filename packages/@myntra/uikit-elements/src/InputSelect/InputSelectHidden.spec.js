import React from 'react'
import { mount } from 'enzyme'

import InputHidden from './InputSelectHidden'

it('should render select element', () => {
  const options = [1, 2]
  const inputHidden = mount(<InputHidden values={options} />)

  expect(inputHidden).toMatchSnapshot()
})
