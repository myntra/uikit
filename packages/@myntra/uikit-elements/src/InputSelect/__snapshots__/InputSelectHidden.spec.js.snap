// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render select element 1`] = `
<InputSelectHidden
  values={
    Array [
      1,
      2,
    ]
  }
>
  <select
    hidden={true}
    multiple={true}
    readOnly={true}
    value={
      Array [
        1,
        2,
      ]
    }
  >
    <option
      key="1"
      value="1"
    />
    <option
      key="2"
      value="2"
    />
  </select>
</InputSelectHidden>
`;
